zebra/zserv.c

```diff
@@ -1452,11 +1455,19 @@ zread_ipv4_route_ipv6_nexthop_add (struct zserv *client, u_short length, struct
 	  switch (nexthop_type)
 	    {
 	    case NEXTHOP_TYPE_IPV6:
-	      stream_get (&nexthop, s, 16);
-              if (nh_count < multipath_num) {
-	        nexthops[nh_count++] = nexthop;
-              }
-	      break;
+              stream_get (&nhop_addr, s, 16);
+              if (nh_count < MULTIPATH_NUM)
```

Comment:
I think we should use `multipath_num` here instead of `MULTIPATH_NUM`.

Please check commit 37fe7731.